### ISSUE TITLE
Aggregating databases with different number of electrodes

### ISSUE THREAD

**Issue by apavlo89** [atpoulsen/Microstate-EEGlab-toolbox#5]
Hello,

I'm new to microstate EEG analysis on 64 channel EEG data and I've tried to follow the steps outlined in your tutorial. I was unable to select multiple EEG recordings because for some of the participants I had to remove 1-2 electrodes that were very noisy. Hence the Microstate toolbox gives me an error because the electrode numbers don't match up.

I've got two questions

1) Is there a workaround to that error so that the code runs for all data regardless of electrode mismatches?
2) In case I can't do 1) what do you suggest I do? Use electrodes common in all participants?

Thank you for your help in the matter

**Comment by apavlo89** [comment]
After some reading, I've realised that the electrodes have to be the same. I've now tried selecting electrodes that are common to all participants. I've used Khanna Arjun Khanna, Pascual-Leone and Farzan (2014) montage of 8 electrode locations (F3/4, C3/4, Cz, P3/4, Pz) that was found to be still optimal for microstate analysis.  (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4257589/). 

Are you aware of any other montages that have been used and that are considered valid I've had to throw quite a lot of participants to get the above derivations.
